CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [1.4.1] 2022-11-08
```

Comment:
well, I was thinking that you add it under the 1.4.0 version until it's released to master.
It's not a bugfix either, so this version is not correct.

So basically just add the added line into the 1.4.0 version and that's it (also maybe fix up the 1.4.0 formatting, because it has no ### Added heading tag.

Basically use the 1.4.0 version as if it was saying that it's [Unrleleased].
We do this approach on repos that don't have dev environments at all.